Gemfile

```diff
@@ -10,3 +10,8 @@ gem 'tomlrb'
 gem 'rake'
 gem 'stove'
 gem 'community_cookbook_releaser'
+gem 'chefspec'
```

Comment:
These should stay out of the gemfile. It's assumed you're testing via ChefDK and don't need gem installs. We were spending a lot of time just updating these files over and over again.